marimbabot_hardware/CMakeLists.txt

```diff
@@ -0,0 +1,207 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(marimbabot_hardware)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
```

Comment:
please cleanup unused comments. The whole file can be ~15 lines long.
Also you did not include `install` rules for the driver binary you build.